### DISCUSSION TITLE
Custom dynamic name PDF template

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#661] [General]
I added a PDF invoice to my booking confirmed emails. It works fine and looks good. But I want to change the name from 
'booking-xx.pdf' to 'Factuur-#_BOOKINGCREATIONDATE{Y}#-PAYMENTID'. Is that possible and how do I do that? I tried adding a code snippet using 
`$dompdf->stream('document.pdf');`
but that snippet was not accepted.

**Comment by liedekef** [comment]
Currently changing the name of the attachment is not possible. It is doable, but requires changes in the templates and email code sections of EME (template: so you can define a name-layout using placeholders and email to actually interpret these and create the name of the attachemt). Feel free to make a donation to make this into an implemented feature :-)

**Comment by JanineKall** [comment]
How much of a donation would make such a thing possible?

**Reply by liedekef** [reply]
contact me via mail for this please :-)
